### ISSUE TITLE
Readme tutorial incorrect

### ISSUE THREAD

**Issue by conmeehan** [jodyphelan/malaria-profiler#4]
Hi,

In the readme file it says the command to run is 
malaria-profiler -1 </path/to/reads_1.fq.gz> -2 </path/to/reads_2.fq.gz> -p <sample_name> -t [threads] --txt 

But this should be `malaria-profiler profile <etc>`

C
